features/support/tag_finder.rb

```diff
@@ -0,0 +1,21 @@
+# Finds Git tags, returns them as TagList instances
+class TagFinder
+
+  # Returns all tags (local + remote)
+  def self.all_tags
```

Comment:
I'd prefer this to just return the TagList and the user can call to_table if thats what they want to do